spec/rubocop/cop/ezcater/style_dig_spec.rb

```diff
@@ -4,7 +4,7 @@
 RSpec.describe RuboCop::Cop::Ezcater::StyleDig, :config do
   subject(:cop) { described_class.new(config) }
 
-  let(:ruby_version) { 2.3 }
+  let(:ruby_version) { 2.6 }
```

Comment:
What prompted this change?

Reply:
Not gonna lie, I couldn't completely figure this one out.  At least not without spending more time on it.

What I think is going on is that this cop had a minimum ruby version that it applied to.  The test was making assertions about the style given a particular version.  `ruby_version` isn't mentioned anywhere else in the file, so I assume it's provided by rubocop, but that's where I stopped digging in.  

I figured that since the gem required 2.6 or higher, we'd never be looking at a ruby lower than that.  Bumping the version here fixed the spec.